qiskit/algorithms/optimizers/nlopts/__init__.py

```diff
@@ -16,39 +16,14 @@
 Installing NLopt
 ================
 
-In order to use any of the NLOpt based global optimizers NLopt must be installed.
-The `NLopt download and installation instructions
-<https://nlopt.readthedocs.io/en/latest/#download-and-installation>`__
-describe how to do this.
-
-If you running on Windows, then you might want to refer to the specific
-`instructions for NLopt on Windows
-<https://nlopt.readthedocs.io/en/latest/NLopt_on_Windows/>`__.
-
-If you are running on a Unix-like system, first ensure that your environment is set
-to the Python executable for which the Qiskit package is installed and running.
-Now, having downloaded and unpacked the NLopt archive file
-(for example, ``nlopt-2.4.2.tar.gz`` for version 2.4.2), enter the following commands:
-
-.. code:: sh
-
-    ./configure --enable-shared --with-python
-    make
-    sudo make install
-
-The above makes and installs the shared libraries and Python interface in `/usr/local`.
-To have these be used by Qiskit, the following commands can be entered to augment the dynamic
-library load path and python path respectively, assuming that you choose to leave these
-entities where they were built and installed as per above commands and that you
-are running Python 3.6:
+In order to use any of the NLopt based global optimizers, NLopt must be installed.
+The following command installs it.
 
 .. code:: sh
 
-    export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib64
-    export PYTHONPATH=/usr/local/lib/python3.6/site-packages:${PYTHONPATH}
+    pip install nlopt
 
-The two ``export`` commands above can be pasted into the ``.bash_profile`` file in the user's
-home directory for automatic execution.  Now you can run Qiskit and these optimizers should be
-available for you to use.
+Further information can be found at
+`NLopt Documentation <https://nlopt.readthedocs.io/en/latest/>`__.
```

Comment:
I think we can just remove this whole docstring now?  We wouldn't have (say) an "Installing scikit-learn" section in our documentation because it's available on pip, and since NLopt is available that way now too, we can probably just remove this entirely (and modify whatever docstring in turn pulls this one in).

If a user doesn't have NLopt installed and tries to use this functionality, the failure message from the lazy-loader will point them in the right direction anyway, I think.